tests/end_to_end/cucumber/steps/steps.py

```diff
@@ -51,20 +56,28 @@ def run_outer_loop(context, n):
     os.chdir(old_cwd)
 
 
-@then("the simulation takes less than {seconds} seconds")
+@when('I run antares-xpansion in memory with the {method} method and {n:d} proc(s)')
+def run_antares_xpansion(context, method, n):
+    command = build_launch_command(context.study_path, method, n, True)
+    process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.DEVNULL, shell=True)
+    out, err = process.communicate()
+    context.return_code = process.returncode
+    context.outputs = read_outputs(Path(parse_output_folder_from_logs(out)) / "expansion" / "out.json")
```

Comment:
why  bother with parsing logs? --> read_outputs(Path("..") / "expansion" / "out.json")
+ the output file is not logged in all steps